support-frontend/assets/pages/[countryGroupId]/checkout.tsx

```diff
@@ -699,6 +712,8 @@ function CheckoutComponent({ geoId }: Props) {
 				supportAbTests,
 				debugInfo: '',
 			};
+			setErrorMessage(undefined);
```

Comment:
Is `undefined` better than explicitly checking for empty string `''` later down here?

Reply:
The type for `errorMessage` is `string | undefined`.

`undefined` is representative of "No error".

This is essentially saying "Unset the errors". If we past a string it would be saying, "Set the error to an empty message".

We're not really checking here - but rather setting.